postgres/process.go

```diff
@@ -0,0 +1,91 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+type Process struct {
+	// Identifier of the process on database.
+	ID uint64 `db:"id"`
+	// The identifier of the process. It follows the parttern 0000000-00.2024.8.26.0000.
+	Number string `db:"number"`
+	// Is the title of the process on esaj (?)
+	Title string `db:"title"`
+	// The status of the process. Possible values are 'active' and 'inactive'.
+	Status ProcessStatus `db:"status"`
+	// Also known as "Cliente/Pasta"
+	CustomerName string `db:"customer_name"`
+	// Also known as "Ação"
+	Action string `db:"action"`
+	// Also known as "Foro"
+	Forum string `db:"forum"`
+	// Date of last movimentation on the process. The time part is 00:00:00 UTC.
+	LastMovimentationDate time.Time `db:"last_movimentation_date"`
+}
+
+type ProcessStatus string
+
+const (
+	ActiveStatus   ProcessStatus = "active"
+	Inactivestatus ProcessStatus = "inactive"
+)
+
+type ProcessRepository interface {
```

Comment:
Does make sense to have this interface on this file or it should be somewhere else?